src/Service/FormAnnotationBuilderFactory.php

```diff
@@ -96,7 +96,7 @@ private function injectFactory(
         AnnotationBuilder $annotationBuilder
     ) {
         if ($formElementManager instanceof FormElementManagerV2Polyfill) {
-            $formElementManager->injectFactory($annotationBuilder, $container);
+            $formElementManager->injectFactory($annotationBuilder, $formElementManager);
```

Comment:
The v3 implementation is therefore passing the real container.
You totally irritated me on this now 🙈